### PR TITLE
[WIP] Bowman lab simulations in FAH pipeline

### PR DESCRIPTION
**Pull request by Mickdub**

### CODE REVIEW

**Comment by hannahbrucemacdonald** [comment]
Just to comment - we have set up the swiss models already:

- SARS-CoV-2 NSP9 DIMER (`swiss model 01`) 
- SARS-CoV-2 NSP12 (RNA-Dependent RNA polymerase) (`swiss model 01`)

Might be worth checking if there are any notable differences between the equilibrated systems. 

Nsp9 now seems to have a crystal structure : `6w4b`

**Comment by Mickdub** [comment]
Yeah, we have some duplicates for sure. We're running FAST on the SARS-CoV-2 proteins and we wanted to use Amber03, which is different from what you're using, so I figured it's still worth putting up. Also, from the FAH slack, it seemed like you weren't necessarily going ahead with putting any of the swiss models on FAH, so I put it here to communicate that we are. We are doing our best to prevent overlap, but it will inevitably happen.

File: system-preparation/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Note that these input files were rapidly prepared using available tools in order
 
 ## Projects
 
-SARS-CoV and SARS-CoV-2 are very similar viruses, so we are using the available structural data from SARS-CoV (both separate simulations and homology-derived models) to aid in understanding of potential SARS-CoV-2 drug targets.
+SARS-CoV and SARS-CoV-2 are very similar viruses, so we are using the available structural data from both viruses (both separate simulations and homology-derived models) to aid in finding potential SARS-CoV-2 drug targets, as well as, understanding why SARS-CoV-2 is fundamentally different from SARS-CoV.
```
Comment (jchodera) [inline system-preparation/README.md]: I'm not sure "finding potential drug targets" is correct, since the proteins are the targets!

Comment (Mickdub) [reply system-preparation/README.md]: Maybe "finding druggable sites on SARS-CoV-2 proteins".  Just wanted to make the point that we're actively looking for pockets to drug the protein targets and additionally we're trying to do some basic science to understand how CoV-1 and CoV-2 are different.

Comment (jchodera) [reply system-preparation/README.md]: That's perfect, thanks @Mickdub !

**Pull request merged**